addons/mail/static/src/components/activity_box/activity_box.scss

```diff
@@ -10,6 +10,11 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ActivityBox_titleLine {
-    border-top: $border-width $border-color dashed;
+.o_ActivityBox_title {
+    //this variable is necessary for the full width chatter margin in mass_mailing
+    margin-top: var(--ChatterActivityBox_title-margin, #{map-get($spacers, 4)});
```

Comment:
Can just name `--ActivityBox-title-margin` (remove `Chatter`, it's simpler to read and specific enough)